lib/database_dumper.rb

```diff
@@ -1205,6 +1203,274 @@ def self.actions_to_column_sanitizers(columns_by_action)
     }.freeze,
   }.freeze
 
+  V2_RESULTS_SANITIZERS = {
```

Comment:
General remark: If the `source_table` is equal to the key of the sanitizer, then you don't have to override

Reply:
I know all these `source_table`s are probably coming from the first draft of the PR when the names were indeed different, but now it's time to clean up :P